src/DefaultPlayer/DefaultPlayer.js

```diff
@@ -122,17 +122,26 @@ DefaultPlayer.propTypes = {
     video: PropTypes.object.isRequired
 };
 
+function isLoading(readyState, networkState, userAgent) {
+  if (readyState == 0 && networkState == 1) {
```

Comment:
Please use strict equality operator ===

Reply:
Thanks for the review!

Yes, this could use `===` though the eslint config of this project doesn't include the [eqeqeq](https://eslint.org/docs/rules/eqeqeq) rule stating it must, and in practice callers are using the browser enums not `null` or other complicating values.